public/js/main.js

```diff
@@ -68,7 +74,7 @@ function hasSelectedTab() {
  */
 function getSelectedTab(tabs) {
   const childId = window.location.hash.split("=")[1];
-  return tabs.find(tab => tab.get("actor").includes(childId));
+  return tabs.find(tab => tab.get("id").includes(childId));
```

Comment:
There were two functional changes to how Tabs work:

1) the top level tab state uses `id` instead of `actor`
2) when a tab is selected either in the UI or URL in main, tab is determined by checking the firefox and chrome field. I think there are some obvious drawbacks here, but am reserving judgement until I see a couple other usecases